sqlalchemy_bigquery/__init__.py

```diff
@@ -16,22 +16,55 @@
 # COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+from .version import __version__
 
-import mock
+from .base import BigQueryDialect
+from .base import (
+    STRING,
+    BOOL,
+    BOOLEAN,
+    INT64,
+    INTEGER,
+    FLOAT64,
+    FLOAT,
+    TIMESTAMP,
+    DATETIME,
+    DATE,
+    BYTES,
+    TIME,
+    RECORD,
+    NUMERIC,
+    BIGNUMERIC,
+)
 
+__all__ = [
+    "__version__",
+    "BigQueryDialect",
+    "STRING",
+    "BOOL",
+    "BOOLEAN",
+    "INT64",
+    "INTEGER",
+    "FLOAT64",
+    "FLOAT",
+    "TIMESTAMP",
+    "DATETIME",
+    "DATE",
+    "BYTES",
+    "TIME",
+    "RECORD",
+    "NUMERIC",
+    "BIGNUMERIC",
+]
 
-def test_dry_run():
-
-    with mock.patch("pybigquery._helpers.create_bigquery_client") as create_client:
-        import pybigquery.api
-
-        client = pybigquery.api.ApiClient("/my/creds", "mars")
-        create_client.assert_called_once_with(
-            credentials_path="/my/creds", location="mars"
-        )
-        client.dry_run_query("select 42")
-        [(name, args, kwargs)] = create_client.return_value.query.mock_calls
-        job_config = kwargs.pop("job_config")
-        assert (name, args, kwargs) == ("", (), {"query": "select 42"})
-        assert job_config.dry_run
-        assert not job_config.use_query_cache
+try:
+    import pybigquery
```

Comment:
Looks like we need to tell `flake8` to ignore this line.

Reply:
fixed